Infographics page w/ factoid

qjhart opened the pull request:
This issue is a starting point for the factoid best practice.

- [x] Review naming convention - Call it ucd-best-practice vs ucd-theme?
- [x] Review imports - Waiting on @spelkey-ucd and SASS libraries
- [ ] Need help on the non-font-awesome icon strategy.

qjhart commented:
@spelkey-ucd - If you have a better naming convention, feel free to let me know.

sbagg commented:
@spelkey-ucd still have questions on:

- [ ] Naming Conventions of Brackets
- [ ] Where Icon Factoids will be placed

qjhart commented:
@sbagg / @spelkey-ucd I've updated this to a infographics page, and added an example web-component representation.   Could maybe use an `<icon>` example but I think it's ready to go.  Let me know what you think.
